GOVERNANCE.md

```diff
@@ -0,0 +1,146 @@
+3) They spend that time doing whatever needs to be done, not necessarily what
+is the most interesting or fun.
+
+Maintainers are often under-appreciated, because their work is harder to appreciate.
+It's easy to appreciate a really cool and technically advanced feature. It's harder
+to appreciate the absence of bugs, the slow but steady improvement in stability,
+or the reliability of a release process. But those things distinguish a good
+project from a great one.
+
+## Reviewers
+
+A reviewer is a core role within the project.
+They share in reviewing issues and pull requests and their LGTM counts towards the
+required LGTM count to merge a code change into the project.
+
+Reviewers are part of the organization but do not have write access.
+Becoming a reviewer is a core aspect in the journey to becoming a maintainer.
+
+## Adding maintainers
+
+Maintainers are first and foremost contributors that have shown they are
+committed to the long term success of a project. Contributors wanting to become
+maintainers are expected to be deeply involved in contributing code, pull
+request review, and triage of issues in the project for more than three months.
+
+Just contributing does not make you a maintainer, it is about building trust
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.
+
+Periodically, the existing maintainers curate a list of contributors that have
+shown regular activity on the project over the prior months. From this list,
+maintainer candidates are selected and proposed on the maintainers mailing list.
+
+After a candidate has been announced on the maintainers mailing list, the
+existing maintainers are given five business days to discuss the candidate,
+raise objections and cast their vote. Votes may take place on the mailing list
+or via pull request comment. Candidates must be approved by at least 66% of the
+current maintainers by adding their vote on the mailing list. The reviewer role
+has the same process but only requires 33% of current maintainers. Only
+maintainers of the repository that the candidate is proposed for are allowed to
+vote.
+
+If a candidate is approved, a maintainer will contact the candidate to invite
+the candidate to open a pull request that adds the contributor to the
+MAINTAINERS file. The voting process may take place inside a pull request if a
+maintainer has already discussed the candidacy with the candidate and a
+maintainer is willing to be a sponsor by opening the pull request. The candidate
+becomes a maintainer once the pull request is merged.
+
+## Adding sub-projects
+
+Similar to adding maintainers, new sub projects can be added to OpenEBS
+GitHub organization as long as they adhere to the OpenEBS vision and mission. 
+New projects are discussed in either the Contributor Meeting or the Community 
+slack and requiers atleast 1 maintainer approval.
```

Comment:
Don't think its a good idea to limit this to meetings or slack. Please consider adding mailing list as an option here as well

Reply:
Will include the mailing list .. same response as above.